src/content/docs/apm/apm-ui-pages/monitoring/find-performance-problems-apm-transactions-page.mdx

```diff
@@ -0,0 +1,374 @@
+
+<Callout variant="tip">
+  If a chart's background is light red, this indicates a time period when an alert condition's [**Critical** threshold](/docs/alerts/new-relic-alerts/defining-conditions/define-thresholds-trigger-alert) has been violated. To view the [incident details](/docs/alerts/new-relic-alerts/reviewing-alert-incidents/view-violation-event-details-incidents) in [New Relic Alerts](/docs/alerts/new-relic-alerts/getting-started/introduction-new-relic-alerts), click the chart.
+</Callout>
+
+<CollapserGroup>
+  <Collapser
+    id="txn-type-dropdown"
+    title="Change the displayed transaction type"
+  >
+    If the **Transaction type** dropdown appears above list of transactions, you can select your choice of transaction type. The available types depend on what transactions your application uses.
+  </Collapser>
+
+  <Collapser
+    id="sort-definitions"
+    title="Select the type of performance measurement (Sort by)"
+  >
+    The **Sort by** dropdown above the transaction list allows you to review transactions according to the following measurements.
+
+    <table>
+      <thead>
+        <tr>
+          <th style={{ width: "200px" }}>
+            **Sort option**
+          </th>
+
+          <th>
+            **Definition**
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+            Most time consuming (default)
+          </td>
+
+          <td>
+            The transactions that consume the most total time. This is calculated by multiplying the number of times the transaction is called by the time the transaction took to complete. In other words, a short transaction called a large number of times or a long transaction called just a few times could both appear at the top of this ranking.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Slowest average response time
+          </td>
+
+          <td>
+            The slowest transactions to complete. A transaction could be called only a few times, but if it is one of the longest transactions to finish, it would appear at the top of this ranking.
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            Apdex most dissatisfying
+          </td>
+
+          <td>
+            The transactions with the highest percentages of user dissatisfaction levels for the response time of your web apps and services ([Apdex](/docs/apm/new-relic-apm/apdex/apdex-measuring-user-satisfaction)).
+
+            The [dissatisfaction level](/docs/apm/new-relic-apm/apdex/view-your-apdex-score#apdex-dissat) is the difference between a perfect Apdex score (1.0) and your app's `apdex_t` ("tolerating") score, based on your [Apdex settings](/docs/apm/new-relic-apm/apdex/change-your-apdex-settings). Transactions that have the highest (worst) percentages appear at the top of this ranking.
```

Comment:
```suggestion
            The [dissatisfaction level](/docs/apm/new-relic-apm/apdex/view-your-apdex-score#apdex-dissat) is the difference between a perfect Apdex score (1.0) and your app's Apdex score, based on your [Apdex settings](/docs/apm/new-relic-apm/apdex/change-your-apdex-settings). Transactions that have the lowest (worst) Apdex scores appear at the top of this ranking.
```

There was a bug where the highest Apdex score got sorted to the top of this list, but we fixed that. Worst/lowest score on top.